databricks/koalas/series.py

```diff
@@ -4730,6 +4730,30 @@ def asof(self, where):
         result_series.name = self.name
         return result_series
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a python scalar.
+
+        Returns
+        -------
+        scalar
+            The first element of Series.
+
+        Raises
+        ------
+        ValueError
+            If the data is not length-1.
+
+        Examples
+        --------
+        >>> kser = ks.Series([10])
+        >>> kser.item()
+        10
+        """
+        if len(self) != 1:
```

Comment:
Let's avoid scanning whole data here. You can do this via collecting 2 rows.

Reply:
Thanks!